_docs/additional-features.md

```diff
@@ -41,7 +41,12 @@ POST /v1/account_validations
   "phone_number": "+233000000000", // mobile phone number to query
   "country": "GH",   // Only "GH" is supported for now
   "currency": "GHS", // Only "GHS" is supported for now
-  "method": "mobile"
+  "method": "mobile",
+  "mobile_provider": "vodafone" // Optional and only for Ghana - if provided validation will be based off phone_number-mobile_provider registration check
```

Comment:
Let's add the comments in a separate section below

Reply:
similar to how we do it in the payout methods

Reply:
also no need to really say that it is only in Ghana as we already say that 3 lines above that this is GH only